reference/pipeline/stages/index.md

```diff
@@ -169,6 +169,9 @@ Deploy a Kubernetes manifest YAML/JSON file.
 ### Find Artifacts From Resource
 Find artifacts from a Kubernetes resource.
 
+### Patch (Manifest)
+Patch an existing Kubernetes resource in-place.
```

Comment:
Would you mind expanding this description a little bit? Ideally, talking about the use cases behind the feature, and any important info about how this stage works. (See the bake stage description for a model of the sort of things that are good to include here -- basically explaining why someone would use this stage, and what they should be aware of when they do so).

The reason that some of these descriptions are only a sentence is that this doc is still a work in progress, and I decided it was more useful to get a skeleton out there than to delay it until it was 100% complete. I'd like to fully explain new stages that are added, though.

Thanks!

Reply:
Done.